ports/rmqcpp/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO bloomberg/rmqcpp
-    REF 52e583c2eb3f06707ce5fde7f29089b17428f6b2
-    SHA512 c4232f37e7bc61d0cc77c52a1283fefc1a7925b4ef8cfbe9b404393540c773feabdc2ad3adc263beb06fe079cbf22010bf1551e15eb2f55b1347f6d8c8f35f8b
+    REF 11859eb470f31008da522b59e96899585b4e94ce
```

```suggestion
    REF 11859eb470f31008da522b59e96899585b4e94ce
```
Could you please explain in detail why you want to update to this commit? Is there any explanation or modification from upstream?

---

- #38736 
 - https://github.com/bloomberg/rmqcpp/commit/43651aeb03776c71df67c7105dfeb78fc5d1b12f

The currently set REF ("old REF") points to a commit that fails building tests, and thus fails building through `vcpkg` altogether, rendering this port unusable.
The REF I've set in this PR ("new REF") fixes that.

They use `libbde`'s `bsl::string` instead of `std::string` to handle `basic_string`-like data.
In one of their test suites, they call a constructor which has the following signature:
```cpp
  basic_resolver_entry(const endpoint_type& ep,
      BOOST_ASIO_STRING_VIEW_PARAM host, BOOST_ASIO_STRING_VIEW_PARAM service)
    : endpoint_(ep),
      host_name_(static_cast<std::string>(host)),
      service_name_(static_cast<std::string>(service))
  {
  }
```
The `BOOST_ASIO_STRING_VIEW_PARAM` macro evaluates to either `std::string_view` or `std::string` depending on the target standard. The former is incompatible with `bsl::string`.
A fix has been applied on the `rmqcpp` side, but the port has not been updated to reflect that.

Since the `README.md` on the repository for the lib mentions `vcpkg` as the primary prerequisite for building `rmqcpp`, old REF being the only available port renders the library unusable.

I hope that clarifies the purpose of this PR. :smiley:

---

Please disable tests in the vcpkg port.